parsl/monitoring/remote.py

```diff
@@ -7,12 +7,10 @@
 from typing import Any, Callable, Dict, List, Sequence, Tuple
 
 from parsl.monitoring.message_type import MessageType
-from parsl.monitoring.radios import (
-    FilesystemRadioSender,
-    HTEXRadioSender,
-    MonitoringRadioSender,
-    UDPRadioSender,
-)
+from parsl.monitoring.radios.base import MonitoringRadioSender
+from parsl.monitoring.radios.filesystem import FilesystemRadioSender
+from parsl.monitoring.radios.htex import HTEXRadioSender
+from parsl.monitoring.radios.udp import UDPRadioSender
```

Comment:
Is there any (ergonomic) value in importing these to `radios/__init__.py` and/or sticking them in the `__all__`?

Reply:
They should go away entirely Real Soon -- they're needed for an `if`-statement that turns a text string radio name into a `..RadioSender`. That enumerates all known radios - both at the imports and in the `if` statement.

Part of my ongoing work here is to turn that into more object style pluggability so that the remote code will receive an object (probably a `...RadioSender`) as part of the task submission, rather than receiving a text string.